front/app/api/comments/useUpdateComment.ts

```diff
@@ -19,7 +19,7 @@ const useUpdateComment = ({
   initiativeId?: string;
 }) => {
   const queryClient = useQueryClient();
-  return useMutation<IComment, CLErrors, IUpdatedComment>({
+  return useMutation<IComment, CLErrorsWrapper, IUpdatedComment>({
```

Comment:
@IvaKop Just a question - I thought that the json wrapped errors were only part of non-React Query hooks? Almost all the other new hooks use the regular CLErrors format at least. What makes the Comments endpoint different? Is it just that the BE hasn't been updated? :thinking: 

![image](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/d407fddb-9ab2-4b21-85c4-46aab3db9017)

Reply:
The issue is that the backend does not always return the errors in the exact same format so sometimes the accurate type is `CLErrors` and sometimes, it's `CLErrorsWrapper` like here. In any case, there is no `json.errors` in either of those types - it's only part of the `CLErrorsJSON`. I know this is confusing, I'll try to clean up those types a bit at the last stage of the conversion.